azurelinuxagent/ga/update.py

```diff
@@ -491,45 +496,93 @@ def _check_threads_running(self, all_thread_handlers):
                 logger.warn("{0} thread died, restarting".format(thread_handler.get_thread_name()))
                 thread_handler.start()
 
+    errors = 0
```

Comment:
where is this used?

Reply:
thanks; left-over debugging code. Removing it

Reply:
thanks; updated